src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidatorTest.java

```diff
@@ -433,7 +433,7 @@ void testErrorReportedWhenRegistrationNumberFieldContainsInvalidCharacters() {
     @Test
     void testNoErrorReportedWhenPublicRegisterNameAndJurisdictionFieldIsMaxLength() {
         entityDto.setOnRegisterInCountryFormedIn(true);
-        entityDto.setPublicRegisterName(StringUtils.repeat("A", 79));
+        entityDto.setPublicRegisterName(StringUtils.repeat("A", 78));
```

Comment:
Could we run this with 79 characters as this would be the edge case

Reply:
But wouldn't this move it up to 160 when you include the comma?

Reply:
yep, thats the edge case as long as it's not >=161 or > 160 it should pass.